plugin.js

```diff
@@ -25,6 +33,21 @@ function fastifyCookieSetCookie (reply, name, value, options, signer) {
   }
 
   const serialized = cookie.serialize(name, value, opts)
+
+  if (opts.enableWarnOnSafeLimit !== false) {
+    const limit = opts.warnOnSafeLimit || 4096
+
+    const bytes = calculateCookieSize(serialized)
```

Comment:
I believe you are trying to calculate the below one?
```suggestion
    const bytes = Buffer.byteLength(name) + Buffer.byteLength(encodeURIComponent(value))
```

Reply:
Yeah that's what I am calculating - tried it with `name` and `value` straight up - but that was too small - so I went for above approach as I couldn't figure out what happened with `value` inside the `serialize` function - I'll try this instead

Reply:
Changed it in latest commit